### ISSUE TITLE
Add eldoc support

### ISSUE THREAD

**Issue by dakra** [emacs-pe/crontab-mode#1]
Would be nice if while entering an entry eldoc would show something like:
```
# min   hour    day     month   day-of-week command
#(0-59) (0-23)  (1-31)  (1-12)  (0-6)
```

I guess that must be set using `eldoc-documentation-function`.

**Comment by dakra** [comment]
Thanks for working on this.
I just tested it and it does show a hint in the minibuffer but only *after* I typed something and only for the current argument.

I like the "normal" eldoc flow better that shows you *all* parameters of the function and just highlights the one you're currently on.

(But don't spend too much time working on it, I rarely edit cron files and your mode is already good enough for those edits. Thanks :)
